xml/art_sle_ha_pmremote.xml

```diff
@@ -123,17 +141,13 @@
      </listitem>
     </varlistentry>
     <varlistentry>
-     <term>Remote Node</term>
+     <term>Physical Remote Node</term>
      <listitem>
       <para>
-       <remark>toms 2017-04-20: yangao: Technically, it's a cluster node that
-        runs the special resource ocf:pacemake:remote which manages the
-        communication with the remote node. Again, if we keep this term here,
-        we'll have to introduce the scenario "baremetals as remote nodes" ;-)</remark>
-       A physical machine that runs the &pmremote; daemon.
-       A remote node runs a special resource (<systemitem
-        >ocf:pacemaker:remote</systemitem>) that
-       manages communication with the cluster.
+       A physical machine that runs the &pmremote; daemon and a special
+       resource (<systemitem>ocf:pacemaker:remote</systemitem>) that
+       manages communication with the cluster (see use case&nbsp;1 in <xref
```

Comment:
A physical machine that runs the &pmremote; daemon. A special resource (<systemitem>ocf:pacemaker:remote</systemitem>) is required to be running on one of the cluster nodes to manage communication between the cluster node and the remote node.